index.js

```diff
@@ -26,7 +26,55 @@ screenTest(mediaqueryList);
 
 mediaqueryList.addListener(screenTest);
 
-const { email } = form.elements;
+let inputData = {};
+if (localStorage.savedForm) {
+  inputData = JSON.parse(localStorage.getItem('savedForm'));
+}
+
+fullName.addEventListener('change', () => {
+  inputData.fullName = fullName.value;
+});
+
+firstName.addEventListener('change', () => {
+  inputData.firstName = firstName.value;
+});
+
+lastName.addEventListener('change', () => {
+  inputData.lastName = lastName.value;
+});
+
+email.addEventListener('change', () => {
+  inputData.email = email.value;
+});
+
+textArea.addEventListener('change', () => {
+  inputData.textArea = textArea.value;
+});
```

Comment:
- [ ] It would be nice if your `inputData` object to have the input fields data ad then pass the entered data into it and then store those data to localStorage so it can be fetched on reload.

- Hint:
```mdx
const inputData = {
    name: document.getElementById('user-name').value,
    email: document.getElementById('user-email').value,
    message: document.getElementById('user-message').value,
  };
```
With that, you'll now populate your localStorage with the inputData.
Then, once your localStorage has the data stored, you'll fetch them now to populate the input fields.

- [ ] Kindly store the input fields data to localStorage. Your project is not persisting data to localStorage at the moment. Please fix that 👍

Reply:
Thank you so much sir for your kind review is it okay? now I have committed the changes